src/bin/main.rs

```diff
@@ -86,3 +81,22 @@ fn main() -> Result<(), Error> {
 
     Ok(())
 }
+
+fn log_setup(config: &ServiceConfig) {
+    let mut env_log_builder = env_logger::builder();
+
+    if let Some(level) = config.core_settings.log_level {
+        env_log_builder.filter_level(level);
+    }
+    if cfg!(feature = "systemd-daemon") {
+        // Logs viewed with journalctl already contain timestamps.
+        env_log_builder.format_timestamp(None);
+    } else {
```

Comment:
I think you can just remove this case and only keep the `else` part, that way we can completely remove this flag 😃

Reply:
and then we can also remove it from `Cargo.toml`

Reply:
Done